os/hal/src/hal_usbh.c

```diff
@@ -338,7 +338,7 @@ usbh_urbstatus_t usbhBulkTransfer(usbh_ep_t *ep,
 
 	osalDbgCheck(ep != NULL);
 	osalDbgCheck((data != NULL) || (len == 0));
-	osalDbgAssert(ep->type == USBH_EPTYPE_BULK, "wrong ep");
+	// osalDbgAssert(ep->type == USBH_EPTYPE_BULK, "wrong ep");
```

Comment:
Are you using usbhBulkTransfer as a generic usbhSyncrhonousTransfer? In this case, we should implement a usbhSyncrhonousTransfer function, and let usbhBulkTransfer wrap usbhSyncrhonousTransfer.

If not, why do you remove the assertion?